features/shop/apply_different_taxes_for_variants_with_different_tax_category.feature

```diff
@@ -0,0 +1,24 @@
+@ui-cart
+Feature: Apply different taxes for variants with different tax category
+    In order to pay proper amount when buying goods with variants from different tax categories
+    As a Customer
+    I want to have correct taxes applied to my order
+
+    Background:
+        Given the store is operating on a single "France" channel
+        And there is "EU" zone containing all members of European Union
+        And default currency is "EUR"
+        And default tax zone is "EU"
+        And store has "EU VAT" tax rate of 23% for "Mugs" within "EU" zone
+        And store has "Low VAT" tax rate of 5% for "Cheap Mugs" within "EU" zone
+        And store has a product "PHP Mug" with "Medium Mug" variant priced at "€100.00" and "Large Mug" variant priced at "€50.00"
```

Comment:
The only problem is, that we can't persist product without price... Shall it then be set into ``sharedStorage`` in ``And store has a product "PHP Mug"`` and used in second one?